blivet/dbus/object.py

```diff
@@ -39,6 +40,22 @@ def _init_dbus_object(self):
                          object_path=self.object_path)
 
     @property
+    def present(self):
+        """ Is this object present in blivet's current view? """
+        return self._present
+
+    def set_presence(self, state):
+        """ Indicate whether the object is in blivet's current view. """
+        conn = self.connection
+        self.remove_from_connection()
+        self._present = state
+        self.add_to_connection(conn, self.object_path)
```

Comment:
Yes, I initially hesitated because I'm tired of properties that do too much in the setter/getter but I don't think one really does too much now that it's done.